docs/python/design.md

```diff
@@ -379,13 +376,16 @@ thing.size = -1
 client.update_thing(name='hello', size=4713, thing=thing)
 ```
 
-#### Methods returning collections (paging)
+#### 컬렉션 반환 메서드 (페이징):
 
-Services may require multiple requests to retrieve the complete set of items in large collections. This is generally done by the service returning a partial result, and in the response providing a token or link that the client can use to retrieve the next batch of responses in addition to the set of items.
+서비스는 큰 컬렉션 내의 아이템 전체 세트를 검색하기 위해 여러 요청을 필요로 할 수 있습니다. 이는 일반적으로 서비스가 부분 결과를 반환하고 응답에 클라이언트가 아이템 세트 외에도 다음 배치의 응답을 검색하는 데 사용할 수 있는 토큰 또는 링크를 제공함으로써 이루어집니다.
 
-In Azure SDK for Python cilent libraries, this is exposed to users through the [ItemPaged protocol](#python-core-protocol-paged). The `ItemPaged` protocol optimizes for retrieving the full set of items rather than forcing users to deal with the underlying paging.
+Azure SDK for Python 클라이언트 라이브러리에서는 이를 사용자에게 [ItemPaged](#python-core-protocol-paged) 프로토콜을 통해 제공합니다. ItemPaged 프로토콜은 사용자가 기본 페이징을 처리하는 것이 아니라 아이템 전체 세트를 검색하는 것에 최적화되어 있습니다.
 
-{% include requirement/MUST id="python-response-paged-protocol" %} return a value that implements the [ItemPaged protocol](#python-core-protocol-paged) for operations that return collections. The [ItemPaged protocol](#python-core-protocol-paged) allows the user to iterate through all items in a returned collection, and also provides a method that gives access to individual pages.
+Azure SDK for Python 클라이언트 라이브러리에서는 이를 사용자에게 [ItemPaged]프로토콜(#python-core-protocol-paged)을 통해 제공합니다. `ItemPaged` 프로토콜은 사용자가 기본 페이징을 처리하는 것이 아니라 아이템 전체 세트를 검색하는 것에 최적화되어 있습니다.
+
+{% include requirement/MUST id="python-response-paged-protocol" %} 컬렉션을 반환하는 작업에 대해 [ItemPaged 프로토콜](#python-core-protocol-paged)을 구현하는 값을 반환합니다. [ItemPaged 프로토콜](#python-core-protocol-paged)은 사용자가 반환된 컬렉션의 모든 아이템을 반복하도록 하며, 개별 페이지에 접근할 수 있는 메소드를 제공합니다.
+return a value that implements the [ItemPaged protocol]
```

Comment:
This line needs to be deleted

Reply:
Just to be sure, i want to ask you that "is it correct to delete sentences that exist in the original English document in the korean version?" if so, i'm curious about the reason.